utils.py

```diff
@@ -17,274 +17,546 @@ def log(str, logfile=None, timestamp=False):
 			print(str, file=f)
 
 def _get_mask_wearing(carefulness, simulation_days, rng):
-	return [rng.rand() < carefulness*BASELINE_P_MASK for day in range(simulation_days)]
+	return [rng.rand() < carefulness * BASELINE_P_MASK for day in range(simulation_days)]
 
 
 def _sample_viral_load_gamma(rng, shape_mean=4.5, shape_std=.15, scale_mean=1., scale_std=.15):
-	""" This function samples the shape and scale of a gamma distribution, then returns it"""
-	shape = rng.normal(shape_mean, shape_std)
-	scale = rng.normal(scale_mean, scale_std)
-	return gamma(shape, scale=scale)
```

Comment:
we should probably start linting to avoid these big diffs

Reply:
yeah, teach me that 👍